style.css

```diff
@@ -218,265 +171,61 @@ form ul li {
   line-height: 24px;
   letter-spacing: 0.03em;
   background-color: white;
+  border-radius: 8px;
 }
 
-.head_2 {
-  padding-bottom: 30px;
+.card-details-div div {
+  margin-bottom: 15px;
```

Comment:
- [ ] [OPTIONAL] Good job implementing the changes suggested by the previous reviewer. It would be great if you add more space to the sides of your cards to match the template. 😎️